src/components/common/CardView/CardInfo/CardInfo.jsx

```diff
@@ -33,14 +33,14 @@ const CardInfo = (props) => {
         </div>
         <div className="cardInfo-container__details-right">
           <div className="cardInfo-container-gender">
-            {removeLastString({gender, age})}
+            {removeLastString({ gender, age })}
           </div>
         </div>
       </div>
       <div className="cardInfo-container__details">
         <div className="cardInfo-container__details-left">
-          <div className="cardInfo-container-phone"><IconPhone />{primaryNumber || 'NA'}</div>
-          <div className="cardInfo-container-email"><IconMail />{email || 'NA' }</div>
+          <div className="cardInfo-container-phone"><IconPhone />{<a href={`tel:${primaryNumber}`}>{primaryNumber}</a> || 'NA'}</div>
```

Comment:
Use Hyperlink component here and in other files as well.
```suggestion
          <div className="cardInfo-container-phone"><IconPhone />{<Hyperlink href={`tel:${primaryNumber}`}>{primaryNumber}</a> || 'NA'}</div>
```

Reply:
Done